### ISSUE TITLE
Table `min-width` is too wide for my screen

### ISSUE THREAD

**Issue by JarrettBillingsley** [izik1/gbops#28]
My laptop is 1440x900, and `150em` is just too wide for it.. doing *Inspect Element* and changing `min-width` to `80em` fixes the issue.

(imo the tables are a lot more readable if the cells are closer to squares anyway)

**Comment by izik1** [comment]
I'll try to figure out a way to do this that doesn't cause:
![image](https://user-images.githubusercontent.com/21113608/139561921-90608467-1d93-4b13-8318-619ada165f5e.png) (note the overflow) it's been on my todo list for a long time, but I'm not sure what of my options really work that well:
1. leave as is (status quo)
2. change the name to something shorter (to what though, the `ldh` mnemonic doesn't tell the whole story)
3. make the font smaller (which feels suboptimal, the font is already fairly small)
4. make the cells able to vary in width (which makes the table feel less clean due to the cells being non-uniform)
5. don't use a pre for formatting, allow the lines to be split between "words"

Currently (to me) the most likely is that last one, since it would allow cells to get arbitrarily small (within reason)...

Investigating

**Comment by JarrettBillingsley** [comment]
fwiw 1400px wide fits comfortably on my screen with plenty of room in each cell for its contents. not saying that's what you should make that the new minimum, which just moves the goalposts, but just to have as a data point ;)

**Comment by izik1** [comment]
@JarrettBillingsley Now that I got CI working again (my travis key expired, and I might as well use actions since I publish to GH-pages), care to look if that solves it?
I _did_ set a minimum of 75em for the table because less than that and you probably need to scroll anyway (I might be able to deal with media queries in the future), but otherwise it should handle much smaller screens somewhat responsibly now. If it has to add an extra line the table won't be evenly sized everywhere, but I don't have the css knowledge to fix that currently. Mostly a backend engineer :sweat_smile: Probably something with css grid... Which I've tried to move to a couple times because it Just Makes More Sense:tm:
